Resources/Prototypes/Entities/Structures/Machines/Computers/computers.yml

```diff
@@ -1011,6 +1172,43 @@
     - CargoBounties
     - Cargo
 
+- type: entity
+  parent: BaseComputerAiAccess
+  id: ComputerFundingAllocation
+  name: funding allocation computer
+  description: Terminal for controlling the distribution of funds and pay to departments.
+  components:
+  - type: Sprite
+    layers:
+    - map: ["computerLayerBody"]
+      state: computer
+    - map: ["computerLayerKeyboard"]
+      state: generic_keyboard
+    - map: ["computerLayerScreen"]
+      state: allocate # ALLOCATION !!!
+    - map: ["computerLayerKeys"]
+      state: generic_keys
+    - map: [ "enum.WiresVisualLayers.MaintenancePanel" ]
+      state: generic_panel_open
+  - type: FundingAllocationConsole
+  - type: ActivatableUI
+    key: enum.FundingAllocationConsoleUiKey.Key
+  - type: ActivatableUIRequiresAccess
+  - type: AccessReader
+    access: [["HeadOfPersonnel"]]
```

Comment:
change to captain, hop has a blatant COI because head of service

Reply:
we're john directioning who we want to take on this access

Reply:
i cant see any reason when hop may even be renamed to service director or something in the future

Reply:
direction works in mysterious ways

Reply:
it is now its own access level, TBD on if we want anyone besides captain to have it